internal/cli/cli.go

```diff
@@ -122,3 +124,125 @@ func (opts *deleteOpts) FailMessage() string {
 	}
 	return fallbackFailMessage
 }
+
+type AtlasAlertsConfig struct {
+	*globalOpts
+	event                           string
+	matcherFieldName                string
+	matcherOperator                 string
+	matcherValue                    string
+	metricThresholdMetricName       string
+	metricThresholdOperator         string
+	metricThresholdUnits            string
+	metricThresholdMode             string
+	notificationToken               string // notificationsApiToken, notificationsFlowdockApiToken
+	notificationChannelName         string
+	apiKey                          string // notificationsDatadogApiKey, notificationsOpsGenieApiKey, notificationsVictorOpsApiKey
+	notificationEmailAddress        string
+	notificationFlowName            string
+	notificationMobileNumber        string
+	notificationRegion              string // notificationsOpsGenieRegion, notificationsDatadogRegion
+	notificationOrgName             string
+	notificationServiceKey          string
+	notificationTeamID              string
+	notificationType                string
+	notificationUsername            string
+	notificationVictorOpsRoutingKey string
+	notificationDelayMin            int
+	notificationIntervalMin         int
+	notificationSmsEnabled          bool
+	enabled                         bool
+	notificationEmailEnabled        bool
+	metricThresholdThreshold        float64
+}
+
+func (opts *AtlasAlertsConfig) buildAlertConfiguration() *atlas.AlertConfiguration {
```

Comment:
[nit] small ranaming here
```suggestion
func (opts *AtlasAlertsConfig) newAlertConfiguration() *atlas.AlertConfiguration {
```